Source/CesiumRuntime/Public/CesiumMetadataValue.h

```diff
@@ -859,3 +860,39 @@ class CESIUMRUNTIME_API UCesiumMetadataValueBlueprintLibrary
   static TMap<FString, FString>
   GetValuesAsStrings(const TMap<FString, FCesiumMetadataValue>& Values);
 };
+
+/// Grants access to metadata value types not currently supported in blueprint.
+/// Can be useful in C++ code.
+/// Should be moved to UCesiumMetadataValueBlueprintLibrary if those types
+/// become compatible with blueprints in the future.
```

Comment:
For documentation we use `/** */` style comments. I also suggest some minor word tweaking, but hopefully no surprises.

```suggestion
/**
 * Grants access to metadata value types that are not currently supported in
 * Blueprints. This can be useful in C++ code.
 *
 * These should be moved to UCesiumMetadataValueBlueprintLibrary if those types
 * become compatible with Blueprints in the future.
 */
```